### DISCUSSION TITLE
Welcome to MSG Discussions!

### DISCUSSION THREAD

**Discussion by Vipin-Sharma** [Vipin-Sharma/MSG#3] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by hbelmiro** [comment]
Hi there 👋🏽
I'm Helber Belmiro and I love Open Source. 
I'm currently working as a Software Engineer at Red Hat and I'm contributing to this project in my spare time as a volunteer.
